Implementations/Libuavcan/Platforms/index.md

```diff
@@ -4,10 +4,71 @@ weight: 20
 
 # Libuavcan platforms
 
-These sections are dedicated to officially supported platforms and also provide documentation
+This section is dedicated to the officially supported platforms and also provide documentation
```

Comment:
This whole doc/section seems a bit backwards - it introduces itself as being about platforms, but then talks about drivers, with nothing to relate them. I expect explanation of what makes up a platform/is in a platform port. 

I would also invert the sections so you explain what the supported platforms are first (and possibly define what you mean by "support"). 

Is the POSIX helper something that should belong in the drivers section?